package.json

```diff
@@ -6,13 +6,29 @@
   "version": "0.0.1",
   "private": true,
   "dependencies": {
+    "@types/common-tags": "1.8.0",
+    "@types/enzyme": "3.10.3",
+    "@types/history": "^4.7.3",
+    "@types/jest": "24.0.17",
+    "@types/marked": "^0.6.5",
+    "@types/lodash": "4.14.144",
+    "@types/mark.js": "8.11.4",
+    "@types/moment-timezone": "0.5.12",
+    "@types/node": "12.7.0",
+    "@types/react": "16.8.24",
+    "@types/react-bootstrap": "0.32.20",
+    "@types/react-dom": "16.8.5",
+    "@types/react-router-dom": "4.3.4",
+    "@types/react-transition-group": "4.2.2",
+    "@types/xregexp": "^3.0.30",
```

Comment:
There's a lot of new types being added here which are duplicated from the dev dependencies. Is that intended? I feel like these types only need to be dev dependencies

Reply:
Some of them came from upstream master

Reply:
@hmhealey some of the types are marked as dependencies, should I move to to devDependencies?